src/controllers/mainController.ts

```diff
@@ -234,11 +234,9 @@ export default class MainController implements vscode.Disposable {
     private async sanitizeProfiles(): Promise<void> {
         let userConnections: any[] = this._vscodeWrapper.getConfiguration(Constants.extensionName).get(Constants.connectionsArrayName);
         for (let conn of userConnections) {
-            if (conn.authenticationType !== 'AzureMFA') {
-                if (conn!.azureAccountToken === '') {
-                    // remove azure account token
-                    conn.azureAccountToken = undefined;
-                }
+            if (conn && conn.authenticationType !== 'AzureMFA' && conn.azureAccountToken === '') {
```

Comment:
let's typically separate the vbump PRs from the functional change PRs.  This helps keep the commit history easier to read through.